src/collective/lazysizes/config.py

```diff
@@ -4,4 +4,6 @@
 
 PROJECTNAME = 'collective.lazysizes'
 
+JS = '++resource++collective.lazysizes/lazysizes.js'
```

Comment:
With a name fixed we will have problems when update a version, needing to purge the cache.
I suggest to generate the page template with webpack and use git revision in the name.

Reply:
as mentioned above, this will be fixed on another pull request.